### ISSUE TITLE
Exclude static fields from mapping.

### ISSUE THREAD

**Issue by dlepex** [aaberg/sql2o#275]
```
class Entity {
public long ver;
public static final Comparator<?> VER = ...
}
```
In the sample above static *VER* field -- which should not participate in mapping at all -- somehow "overrides" the instance *ver* field => so the instance field is not set and is always ZERO when selecting from DB.

**Comment by zapodot** [comment]
@dlepex As you can see I have implemented your suggestion. I will leave it to @aaberg to review it.

**Comment by aaberg** [comment]
Resolved by pr #284
